### PR TITLE
feat: Add conversation history selection options to ConditionAgent node

### PR DESCRIPTION
**Pull request by jeanibarz**
Enhance the `ConditionAgent` node by adding conversation history selection options, aligning it with the existing `Agent` and `LLMNode` nodes. This feature allows users to filter the conversation history messages used in the `ConditionAgent`, providing greater control over the context included in prompts.

### Changes

- **Added Parameter:**
  - `conversationHistorySelection`: Allows users to choose which messages from the conversation history to include in prompts.
  
- **Selection Options:**
  - **User Question:** Use the user question from the historical conversation messages as input.
  - **Last Conversation Message:** Use the last conversation message from the historical conversation messages as input.
  - **All Conversation Messages:** Use all conversation messages from the historical conversation messages as input.
  - **Empty:** Do not use any messages from the conversation history. Ensure to use either System Prompt, Human Prompt, or Messages History.
  
- **Default Selection:**
  - Set to `'All Conversation Messages'` to maintain comprehensive context by default.

### Related Issues

- **Previous related PR:** #3653

### CODE REVIEW

File: packages/components/nodes/sequentialagents/ConditionAgent/ConditionAgent.ts [hunk 1]
```diff
@@ -185,6 +187,42 @@ class ConditionAgent_SeqAgents implements INode {
                 additionalParams: true,
                 optional: true
             },
+            {
```
Comment (HenryHengZJ) [inline packages/components/nodes/sequentialagents/ConditionAgent/ConditionAgent.ts:190]: thanks @jeanibarz ! can we increment the `this.version = 2.0` to `3.0`, and have the parameter order as the same in Agent/LLMNode:

1. System Prompt
2. Prepend Messages History
3. Conversation History
4. Human Prompt

**Comment by HenryHengZJ** [comment]
there was some linting issue from the main branch, can you pull the latest changes?

**Comment by jeanibarz** [comment]
Yes I noticed that yerteday: done 👍

**Review by HenryHengZJ — APPROVED** [review]
thanks @jeanibarz !

**Pull request merged**